src/moveShortcuts.asm

```diff
@@ -162,33 +162,33 @@ afterShortcuts:
 ; Prevent passing turn by pressing A+B if the move dialogue box is open
 ; -----------------
 .org EU_22F1D18
-.area 0x22F1D44 - .
+.area 0x30 ; 0x22F1D44 - .
```

Comment:
Why was this changed? I'd rather keep the original format so it's clear where the available area ends.
It's not necessary to make any changes to support other versions, sice the result of the subtraction is the same in all of them.

Reply:
0x22F1D44 - 0x22F295C = -0xC18
Which results in a negative area size error.